c/src/lib.rs

```diff
@@ -310,6 +310,26 @@ pub unsafe extern "C" fn PFCanvasSetLineDash(canvas: PFCanvasRef,
     (*canvas).set_line_dash(slice::from_raw_parts(new_line_dashes, new_line_dash_count).to_vec())
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn PFCanvasSetCurrentTransform(canvas: PFCanvasRef, transform: *const PFTransform2F) {
+    (*canvas).set_current_transform(&(*transform).to_rust());
```

Comment:
This is interesting because it seems like we're ultimately copying the transform twice--once to make it owned by Pathfinder, but then the canvas copies it again to make it its own.  Is there a way to optimize this to only be one copy?  Or will Rust actually be super smart and optimize things at compile time?

Reply:
It shouldn't be a big deal either way. None of the canvas methods are optimized that well to begin with—the performance-critical parts are the scene building.